Analysis/tasks.py

```diff
@@ -71,6 +73,17 @@ class HistProducerFileTask(Task, HTCondorWorkflow, law.LocalWorkflow):
     max_runtime = copy_param(HTCondorWorkflow.max_runtime, 5.0)
     n_cpus = copy_param(HTCondorWorkflow.n_cpus, 1)
 
+    Cache_Module = None
+    def __init__(self, *args, **kwargs):
+        super(HistProducerFileTask, self).__init__(*args, **kwargs)
+        # self.setup = Setup.getGlobal(os.getenv("ANALYSIS_PATH"), self.period, self.customisations)
+
+        if self.Cache_Module == None:
+            self.use_ana_cache = 'analysis_cache_import' in self.setup.global_params
+            if self.use_ana_cache:
+                # self.ana = importlib.import_module(self.setup.global_params['analysis_cache_import'])
+                self.Cache_Module = importlib.import_module(self.setup.global_params['analysis_cache_import'])
```

Comment:
make also cache task name configurable. E.g. something like 
```suggestion
                file, className = self.setup.global_params['analysis_cache_import'].split(':')
                cacheModule = importlib.import_module(file)
                HistProducerFileTask.cacheClass = getattr(cacheModule, className)
```